spinn_front_end_common/interface/interface_functions/locate_executable_start_type.py

```diff
@@ -51,17 +62,27 @@ def __call__(self, graph, placements, graph_mapper=None):
                         vertex, placements,
                         binary_start_types[placement_binary_start_type])
                 elif graph_mapper is not None:
-                    machine_verts = graph_mapper.get_machine_vertices(vertex)
+                    machine_verts = graph_mapper.get_machine_vertices(
+                        vertex)
                     for machine_vertex in machine_verts:
                         self._add_vertex_to_subset(
                             machine_vertex, placements,
-                            binary_start_types[placement_binary_start_type])
+                            binary_start_types[
+                                placement_binary_start_type])
```

Comment:
Don't think this needs splitting across two lines.

Reply:
fixed.